pkg/steps/types/types.go

```diff
@@ -0,0 +1,19 @@
+package steptypes
+
+import (
+	api "github.com/openshift/ci-tools/pkg/api"
+)
+
+type TestStep struct {
+	Name        string
+	Image       string
+	Commands    string
+	ArtifactDir string
+	Resources   api.ResourceRequirements
+}
+
+type TestFlow struct {
```

Comment:
You mentioned this on Slack but just so we don't lose it, we should pass through the cluster profile here